### DISCUSSION TITLE
Permissions for each forum category/post

### DISCUSSION THREAD

**Discussion by ShellarFX** [spatie/laravel-permission#2127] [Q&A]
Hello. I'm trying to create something like a forum. I would like to create moderators for each category on my forum.

I was thinking how can i manage those permissions. For example i have 2 categories: `games`, `computers`. I should have one moderator for games and one for computers categories. I can create roles `moderator.games` and `moderator.computers` with permissions `games.posts.delete`, `computers.posts.delete`, etc.

But is this a good solution? If i will use this not only with categories, but also with posts (`post.{postId}.edit`, etc) there can be thousands of roles and even more permissions.

**Comment by erikn69** [comment]
Maybe you are looking for `silber/bouncer`
